io/plugins/eu.esdihumboldt.hale.io.csv.test/META-INF/MANIFEST.MF

```diff
@@ -7,8 +7,13 @@ Fragment-Host: eu.esdihumboldt.hale.io.csv;bundle-version="2.5.0"
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Import-Package: eu.esdihumboldt.cst.test,
  eu.esdihumboldt.hale.common.core,
+ eu.esdihumboldt.hale.common.instance.groovy,
+ eu.esdihumboldt.hale.common.schema.groovy,
  eu.esdihumboldt.hale.common.test,
+ eu.esdihumboldt.util.groovy.sandbox,
  org.junit;version="4.8.1",
  org.junit.rules;version="4.11.0"
 Bundle-Vendor: data harmonisation panel
 Automatic-Module-Name: eu.esdihumboldt.hale.io.csv.test
+Require-Bundle: eu.esdihumboldt.util.groovy;bundle-version="4.2.0",
+ groovy-all;bundle-version="2.4.14"
```

Comment:
The bundle that should be included as dependency for Groovy should be `groovy` instead of `groovy-all`.